HW1/P5/P5.txt

```diff
@@ -1 +1,4 @@
-# Your discussion here
+One of the shortest paths from Kevin Bacon to Harvard University is: Kevin_Bacon -> College_Bowl -> Harvard_University, where a -> b means "a links to b".
+One of the shortest paths from Harvard University to Kevin Bacon is: Harvard_University -> 1998 -> 1958 -> Kevin_Bacon
+
+My code for connected components was not fast enough to arrive at an answer
```

Comment:
I actually got a different set of answers here. Shortest path from Kevin Bacon to Harvard for me was several steps long (College Bowl went to John Lithgow to Marisa Silver to 6 degrees of separation to Time), and for Harvard to Kevin there were 120 paths.